hrms/hr/doctype/job_applicant/job_applicant.json

```diff
@@ -1,5 +1,6 @@
 {
  "actions": [],
+ "allow_import": 1,
```

Comment:
You will have to modify the `modified` timestamp in JSON to sync the schema on migrate